.travis.yml

```diff
@@ -1,16 +1,17 @@
 language: python
-python: 2.7
+python: 
+  - 2.7
+  - 3.5
 addons:
   firefox: 47.0.1
 env:
-- TOXENV=tests DISPLAY=:99.0
-- TOXENV=flake8
+  - DISPLAY=:99.0
```

Comment:
If you add a matrix of includes, you should get a good split of jobs in Travis:

```yml
env:
  - DISPLAY=:99.0
matrix:
  include:
  - python: 2.7
    env: TOXENV=flake8
  - python: 2.7
    env: TOXENV=marionette
  - python: 3.5
    env: TOXENV=flake8
  - python: 3.5
    env: TOXENV=marionette
```

You should then be able to allow py35-marionette to fail by using:

```yml
matrix:
  ...
  allow_failures:
  - python: 3.5
    env: TOXENV=marionette